docker-compose.yml

```diff
@@ -71,6 +71,9 @@ services:
       - APP_URL=${APP_URL:-http://testserver:8080/blacklab-server}
       - INDEX_TEST_DATA_ROOT=${INDEX_TEST_DATA_ROOT:-data}
     build: test
+    volumes:
+      # We need access to the previous responses and be able to save responses for any new tests.
+      - ./test/saved-responses:/app/saved-responses
```

Comment:
I am curious about why we are mounting the directory as opposed to copying the files to the docker container.

Reply:
If we keep them in the docker container, downloading new test files will be just a matter of adding such files to the remote storage

Reply:
The idea behind the bind mount is that if there was no saved response, the test passes and the response is saved. This means I can add a test, run the script, and commit the test and response right away.

Of course we don't need the bind mount on GitHub, so now there's two scripts: `run-local.sh` that uses the bind mount and saves missing responses, and `run-ci.sh` for GitHub without the bind mount that uses the responses from inside the image (or downloaded in your case) and fails if one is missing.

I'd like to regularly add new tests. If you have any suggestions on how to reduce friction on your side, let me know.

Reply:
great that makes sense, thanks for doing that!